### ISSUE TITLE
Show app descriptions

### ISSUE THREAD

**Issue by chadalen** [linuxappstore/linuxappstore-frontend#10]
Clicking on an app will show it's description. Along with the author, categories, screenshots etc

**Comment by lode** [comment]
Related to this, it would be great if a single app has a single page for all sources. E.g. have a single page for SublimeText which offers a link to flatpak & snap, and shows which of those is officially supported.

**Comment by probonopd** [comment]
This.

**Comment by chadalen** [comment]
@lode I agree as well. The only issue with this is that the data I get from their API's is not standardized.

E.g if you search "wps" to get WPS Office
![https://imgur.com/83BAHll](https://imgur.com/83BAHll)

There's no way for me to tell if this Flatpak spelt "WPS Office" is the same as the snap "wps-office" without manually going into my database and changing that. I'm trying to avoid doing manual changes I want everything to be automated.

I could do this at the view level though and if they have the same name then I could display a single page with the link to the flatpak, snap or appimage there?

**Comment by lode** [comment]
Yeah, I expected something like that.

Personally I would make the bold move and try to force your sources to come to an agreement. If you group apps (data or view level shouldn't matter - as long as the end user understands it as a single entity) based on name, some at least look good, and for the others the owners of apps will start to standardize names. (The same for the icon, just pick one - e.g. the last updated one - and owners can follow.) Of course this needs good documentation what app owners can do to make the grouping work, and that this website becomes a success.

And otherwise, for the maintenance part, you could make something so that the community can group things, update titles and icons, etc. This way you can share the maintenance costs.

I think one or the other way around, the grouping is essential. In my eyes the whole point of this website is to stop bothering about the source. And now when looking through the search results, I still have to.

**Comment by probonopd** [comment]
> Personally I would make the bold move and try to force your sources to come to an agreement. 

For that, there would need to be a clear and unambiguous way how to derive that information from whatever binaries or source code the application authors provide.

**Comment by lode** [comment]
> For that, there would need to be a clear and unambiguous way how to derive that information from whatever binaries or source code the application authors provide.

I'm not sure I understand what you mean. What I meant was to pick the name which is now displayed in the search results, and group applications based on that. Applications which aren't grouped even though they are the same will be a pain in the ass for their end users but also for the owners. By doing so, owners/sources will be forced/seduced to start using the same name everywhere.

**Comment by probonopd** [comment]
Yes, using the name is reasonable but not a guarantee that everything will be consistent from the beginning, e.g., is "VLC" the same as "VideoLAN Client" and "VLC Video Player"?

**Comment by bratting** [comment]
I think, in large parts, Linux app store unusable. If you don’t recognize the icon, the ‘app store’ have no information about the app at all. The information is only about what App image, Flatpack and Snap are.

**Comment by chadalen** [comment]
@bratting I'll add this to 0.4.0 release. I've been wanting to do this for a while now.
